config/serverless.oblt.yml

```diff
@@ -141,11 +51,6 @@ xpack.fleet.agentIdVerificationEnabled: false
 ## Enable event.ingested separately because agentIdVerification is disabled
 xpack.fleet.eventIngestedEnabled: true
 
-## Enable the capability for the observability feature ID in the serverless environment to take ownership of the rules.
-## The value need to be a featureId observability Or stackAlerts Or siem
-xpack.alerting.rules.overwriteProducer: 'observability'
-xpack.observability.createO11yGenericFeatureId: true
```

Comment:
Remove the creation of the generic o11y feature id.

Reply:
I still can see a flag to explicitly disable custom roles in this yml file. https://github.com/elastic/kibana/blob/bba42298efb8bcc103c4c437323fb4a449be6432/config/serverless.oblt.yml#L141

Do you plan to remove it?